typings/index.d.ts

```diff
@@ -2761,6 +2761,7 @@ declare module 'discord.js' {
     roles?: PartialRoleData[];
     systemChannelID?: number;
     verificationLevel?: VerificationLevel | number;
+    systemChannelFlags?: SystemChannelFlagsResolvable;
```

Comment:
Please move this property above verificationLevel so they're sorted alphabetically.